### PR TITLE
Update data-models.md

### PR DESCRIPTION
**Pull request by subatomicat**

### CODE REVIEW

**Comment by subatomicat** [comment]
Hi Alina! I am actually working on this lesson as part of an application for Curriculum Developer at Techtonica. I hope my work is helpful. It is a bit out of my comfort zone, but I learned a lot! I would be happy to work more on it tonight/tomorrow if that timing works for you.

**Comment by alodahl** [comment]
No wonder it was so good!  Tonight or tomorrow would be awesome.  Good luck 🤞🏻

**Comment by subatomicat** [comment]
So good!? Aww shucks. Thanks.

File: databases/data-models.md [hunk 1]
```diff
@@ -1,69 +1,75 @@
...
+
+**Brief outline of slides**
+- Define data, data model/modeling, database
+- Give example of data and how it how its relationship can be shown with a diagram
+- Show different types of data model schemas
+- Give an example of a database model
+- Show an example of what types of data are needed given a user flow
+- Show how that data is organized into a database model
 
 ### Common Mistakes / Misconceptions
```
[the remark this answers is not in the document]
Comment (subatomicat) [reply databases/data-models.md]: I was worried about removing it. I am not a content expert so I thought someone else may have some ideas for this section. But I will delete.

File: databases/data-models.md [hunk 2]
```diff
@@ -1,69 +1,75 @@
...
+	- Star schema
+	- The object-relational model, which combines the two that make up its name
 
 ### Materials
 
-- [This website](example.com)
-- [This other website](otherexample.com)
+- [What is a Database Model (Lucidchart)](https://www.lucidchart.com/pages/database-diagram/database-models#top-info)
+- [Wikipedia: Database model](https://en.wikipedia.org/wiki/Database_model)
+- [Data Modeling - An Introduction](https://www.youtube.com/watch?v=tR_rOJPiEXc)
```
[the remark this answers is not in the document]
Comment (subatomicat) [reply databases/data-models.md]: Thanks. Michelle gave me the first two. The video was the first resource I found that made any sense to me. I am not seeing the data model examples in java. Where did you see them? It seems like this lesson is more of an introduction to data modeling and could be pretty language-agnostic, but I can take a look for more resources.

[the remark this answers is not in the document]
Comment (subatomicat) [reply databases/data-models.md]: I'm sorry. I am not seeing any resources that show javascript data models.

File: databases/data-models.md [hunk 3]
```diff
@@ -1,69 +1,75 @@
...
-
-Make sure they avoid this: thing
+Not sure
 
 
 ### Guided Practice
 
-Have the participants work with you as you do something.
-
+Give participants different types of data and work with them to create a diagram that represents the relationship between the data.
```
[the remark this answers is not in the document]
Comment (subatomicat) [reply databases/data-models.md]: I added a more specific example with some possible models that participants might create.

**Pull request merged**